structure_tests/structure_test_v1.go

```diff
@@ -186,7 +186,21 @@ func ProcessCommand(t *testing.T, envVars []EnvVar, fullCommand []string, checkO
 		} else {
 			t.Fatalf("Error running setup/teardown command: %s.", err)
 		}
-		exitCode = err.(*exec.ExitError).Sys().(syscall.WaitStatus).ExitStatus()
+
+		exitErr, ok := err.(*exec.ExitError)
+		if !ok {
+			exitErr, ok := err.(*exec.Error)
+			if ok {
+				t.Logf("Command %s failed to run! Error: %s", exitErr.Name, exitErr.Error())
+				t.Logf("Attempting to run in shell mode.")
```

Comment:
Yeah I think I'd err towards being explicit rather than implicit. You could drop that part and just document that people can use "sh -c" themselves in the cmd if they need shell behavior, or add a bool to add that implicitly.